packages/orchestrator-ui-components/src/messages/nl-NL.json

```
@@ -47,6 +47,7 @@
             "cancel": "Annuleren",
             "submit": "Verstuur",
             "previous": "Vorige",
+            "previousQuestion": "Weet je zeker dat je deze pagina wilt verlaten? Ingevulde velden worden niet opgeslagen",
```

Nou nou, gaan applicaties nu ook tutoyeren? :stuck_out_tongue: 
```suggestion
            "previousQuestion": "Weet u zeker dat u deze pagina wilt verlaten? Ingevulde velden worden niet opgeslagen",
```

---

For now that seems to be the way, looking at the other translations. Maybe a good idea to revisit this, since `u` might be better indeed.